### DISCUSSION TITLE
A Few Notes

### DISCUSSION THREAD

**Discussion by bryanjordin** [garettB/shopify-flask-example#26] [General]
I thought I would provide a few notes based on my experience testing this app framework. I did my best to follow instructions to the letter and have basic experience building Flask and Shopify applications,  just not together.

**Jinja and escape module support**

The requirements.txt calls for Flask-1.1.2. Jinja is a dependency of Flask and Flask V1.X.X uses the escape module from Jinja, however recently support for the escape module was [dropped in newer versions of Jinja](https://jinja.palletsprojects.com/en/3.1.x/changes/#version-3-1-0).

To fixx, change the requirements.txt doc to use a version of Flask after 2.2.2

**Comment by garettB** [comment]
Hi @bryanjordin , thanks for the comment! I am aware of the issue and know I need to get this updated. I'm hoping to get it done by end of summer but life is busy. If you'd like to submit a tested PR I'd be happy to merge until I get the chance to make this update with a number of other tweaks I'm hoping to add.
